### DISCUSSION TITLE
Add a schedule view/popup

### DISCUSSION THREAD

**Discussion by pjeweb** [alveusgg/extension#21] [Ideas]
![Mockup weekly show schedule](https://user-images.githubusercontent.com/684458/213874691-83c03a10-d6b6-48b3-913e-aa4354054885.png)

- Could show a schedule with weekly shows
- Could also show special events or any other planned content
- Would be nice to show the time both in alveus timezone and user timezone (see mockup)

Problems:
- This would again require a new button to toggle this view
- Kind-of duplicates the Twitch schedule feature (that I do not find as easy to use and is in my experience seldomly used by other twitch communities)

**Comment by abdullahmorrison** [comment]
I like this. I plan on adding a section that just explains what alveus is (see image) so I could just add the schedule with it somehow. How possible this is depends on how clever the design is. I'm hesitant to add a button for it because it doesn't feel important enough to increase the clutter. Gotta sit on it more.

<img src="https://user-images.githubusercontent.com/49528805/213887004-f3fb621d-ad3f-4a4c-9d3e-d71cc23895a9.png" width=500/>
